config/workers.coffee

```diff
@@ -116,13 +174,34 @@ module.exports = (KONFIG, options, credentials) ->
         ]
       healthCheckURLs   : [ "http://localhost:#{KONFIG.kloud.port}/healthCheck" ]
       versionURL        : "http://localhost:#{KONFIG.kloud.port}/version"
+      kubernetes        :
+        image           : 'koding/base'
+        command         : " [ \"./run\", \"exec\", \"#{GOBIN_K8S}/kloud\" ] "
+        mounts          : [ 'koding-working-tree', 'root-kite-volume', 'generated-volume' ]
+        ports:
+          containerPort : "#{KONFIG.kloud.port}"
+          hostPort      : "#{KONFIG.kloud.port}"
```

Comment:
We can eliminate the host port in next iteration, we wont need it when we have service descriptions in place.